src/analyses/base.ml

```diff
@@ -823,7 +824,7 @@ struct
           | [] -> assert false
           | (e1, e2) :: eqs ->
             let eqs_for_all_mem e = List.for_all (fun (e1, e2) -> CilType.Exp.(equal e1 e || equal e2 e)) eqs in
-            let eqs_map_remove e = List.map (fun (e1, e2) -> if CilType.Exp.equal e1 e then e2 else e1) eqs in
+            let eqs_map_remove e = Iter.on_list (Iter.map (fun (e1, e2) -> if CilType.Exp.equal e1 e then e2 else e1)) eqs in
```

Comment:
Does changing a single `List` operation, as here, improve performance?

Reply:
Unfortunately, these single List operations don't improve performance. We added them anyway thinking that maybe flambda can do something clever with that.
I'd suggest they can/should be removed before merging this branch.